src/libstd/sys/unix/ext/raw.rs

```diff
@@ -15,6 +15,8 @@
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type uid_t = u32;
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type gid_t = u32;
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type pid_t = i32;
+#[unstable(feature = "pthread_t", issue = "0")]
+pub type pthread_t = usize;
```

Comment:
In terms of what actually happens they're all the same size (I believe), but I think it's best to ensure that they're the same definition so the libstd pthread_t is interoperable with the libc pthread_t.